wsidicom/geometry.py

```diff
@@ -473,3 +486,25 @@ def __sub__(self, value: PointMm) -> 'RegionMm':
         if isinstance(value, PointMm):
             return RegionMm(self.position - value, self.size)
         return NotImplemented
+
+    def to_other_origin(
+        self,
+        origin: PointMm,
+        orientation: Sequence[float]
+    ) -> 'RegionMm':
+        position = self.position - origin
+        if orientation == [0, 1, 0, 1, 0, 0]:
+            position = PointMm(position.y, position.x)
+            size = SizeMm(self.size.height, self.size.width)
+        elif orientation == [0, -1, 0, -1, 0, 0]:
+            position = PointMm(-position.y, -position.x)
+            size = SizeMm(-self.size.height, -self.size.width)
+        elif orientation == [1, 0, 0, 0, -1, 0]:
+            position = PointMm(position.x, -position.y)
+            size = SizeMm(self.size.width, -self.size.height)
+        elif orientation == [-1, 0, 0, 0, 1, 0]:
+            position = PointMm(-position.x, position.y)
+            size = SizeMm(-self.size.width, self.size.height)
+        else:
+            raise NotImplementedError("Non-implemented orientation")
```

Comment:
It is the library that only implements the typically used orientations. I cant find anything in the standard that restricts the orientation. It might not even have to be Cartesian...